dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/SchedulingHelper.java

```diff
@@ -280,8 +281,9 @@ public void run() {
             if (this.isDone()) {
                 return;
             }
-
-            this.m_defaultFuture = this.m_executor.submit(this.m_callable);
+            ExpeditedFutureTask<V> futureTask = new ExpeditedFutureTask<V>(this.m_callable);
+            this.m_executor.execute(futureTask);
+            this.m_defaultFuture = futureTask;
```

Comment:
I realize the pre-existing code wasn't doing this (it couldn't) but seems like it would be safer to set m_defaultFuture prior to issuing the execute, just in case the execute triggers anything that would expect m_defaultFuture to have already been set.

Reply:
Yep- that would be better, I'll update that in my next PR